airflow/kubernetes/pod_generator.py

```diff
@@ -396,7 +240,7 @@ def from_legacy_obj(obj) -> Optional[k8s.V1Pod]:
                     limits=limits
                 )
         namespaced['resources'] = resources
-        return PodGenerator(**namespaced).gen_pod()
+        return PodGeneratorDeprecated(**namespaced).gen_pod()
```

Comment:
Can/should we issue a FutureDeprecationWarning here, so that we can remove this code path in 2.1/2.2?

Reply:
@ashb there is a future deprecation https://github.com/apache/airflow/pull/10393/files#diff-868ed785b2a336f20cb6a577dde2502aR198-R201

Reply:
Oh, you issue a warning a few lines up (from the previous PR. right?)

Reply:
Yep :)